tests/integration_tests/build/test_clippy.py

```diff
@@ -18,7 +18,11 @@
     TARGETS
 )
 def test_rust_clippy(target):
-    """Fails if clippy generates any error, warnings are ignored."""
+    """
+    Test that clippy does not generate any errors/warnings.
+
+    @type: build
```

Comment:
Can't we just fetch the type from the test path?

Reply:
We can, but we would unnecessarily tie the folder structure to the test report. While I don't see this as a concern for now, we may decide to change it in the future (it could already use some love). WDYT?

Reply:
Makes sense, let's keep it this way.